### ISSUE TITLE
Broken SyncRequestClient.getBalance()

### ISSUE THREAD

**Issue by PavelRavvich** [Binance-docs/Binance_Futures_Java#47]
Always throw 
`[Executing] -1021: Timestamp for this request was 1000ms ahead of the server's time.`

In `ApiSignature` class code is very clear:
```
        builder.putToUrl("recvWindow", Long.toString(BinanceApiConstants.DEFAULT_RECEIVING_WINDOW))
                .putToUrl("timestamp", Long.toString(System.currentTimeMillis()));
```
Part of the lib method works fine like `getExchangeInformation`. 
But `getBalance`, `getAccountInformation` throws this error.
Or do I miss something important?

**Comment by dvaNeba** [comment]
@PavelRavvich, I had the same problem. Fixed by changing the time on my computer from "set manually" to "set automatically".

**Comment by PavelRavvich** [comment]
Another solution is to get time from `https://fapi.binance.com/fapi/v1/time`

**Comment by Angus-Less** [comment]
> @PavelRavvich, I had the same problem. Fixed by changing the time on my computer from "set manually" to "set automatically".

Yes this is a problem that permeates all APIs. The best bet is to either re-sync your computer's clock, or get Binance's system clock at the start of every major function call.

**Comment by itsram90** [comment]
this is a bug, dont try to hide it, systems are developed for common people not for Einstein of the world.  i have faced this issues many times, my system clock is correct, it is set to auto
